backends/ipvs-as-sink/ipvs.go

```diff
@@ -284,6 +284,9 @@ func (s *Backend) SetUpHttpListen() error {
 	return <-errCh
 }
 
+
+// ServeProxyMode used for creating the type of proxy which can be tested by proxyMode in service.go e2e test 
```

Comment:
what do we mean by "type of proxy" here? I've already forgotten why we had to port this in from the original codebase in kubernetes/kubernetes